util/BUILD

```diff
@@ -7,6 +7,7 @@ cc_library(
         "string.h",
     ],
     visibility = ["//visibility:public"],
+    deps = ["@range-v3//:range-v3"],
```

Comment:
Is it possible to use `@ranges-v3//:std` to ensure that we can switch to the stdlib ranges once they're available in all compilers/stdlib combos we support?